VideoAPI/Testing.Common/Assertions/AssertConferenceForAdminResponse.cs

```diff
@@ -18,7 +18,7 @@ public static void ForConference(ConferenceForAdminResponse conference)
             conference.Id.Should().NotBeEmpty();
             conference.HearingRefId.Should().NotBeEmpty();
             conference.Status.Should().NotBeNull();
-            conference.TelephoneConferenceId.Should().NotBeEmpty();
+            //conference.TelephoneConferenceId.Should().NotBeEmpty();
```

Comment:
I dont think theres a need to commit commented code :) we have git for history :)